pkg/rke2/rke2_linux.go

```diff
@@ -64,6 +66,16 @@ func initExecutor(clx *cli.Context, cfg Config, dataDir string, disableETCD bool
 			Name: cfg.CloudProviderName,
 			Path: cfg.CloudProviderConfig,
 		}
+		nodeName := clx.String("node-name")
+		if nodeName == "" && cfg.CloudProviderName == "aws" {
```

Comment:
We only appear to use the nodeName in the test; don't bother creating a variable for it?
```suggestion
		if clx.String("node-name") == "" && cfg.CloudProviderName == "aws" {
```

Reply:
fixed